data/base/common/alp/config.yaml

```diff
@@ -26,6 +26,7 @@ config:
     common-services:
       - boot.device-mapper
       - haveged
+      - NetworkManager
```

Comment:
We also need `transactional-update-cleanup.timer` which I noticed not to be in the SL Micro 6 images compared to SLE Micro 5.5 and `NetworkManager-wait-online`

Reply:
`NetworkManager-wait-online` gets enabled automatically when `NetworkManager.service` is enabled. It seems `transactional-update-cleanup.timer` was removed from the systemd presets, I wonder why. Anyway, easy enough to add.